### ISSUE TITLE
训练分词模型出现问题, crf-loss出现负数

### ISSUE THREAD

**Issue by yongzhuo** [BrikerMan/Kashgari#321]
**You must follow the issue template and provide as much information as possible. otherwise, this issue will be closed.
请按照 issue 模板要求填写信息。如果没有按照 issue 模板填写，将会忽略并关闭这个 issue**

## Check List
Thanks for considering to open an issue. Before you submit your issue, please confirm these boxes are checked.

**You can post pictures, but if specific text or code is required to reproduce the issue, please provide the text in a plain text format for easy copy/paste.**

- [ ] I have searched in [https://github.com/BrikerMan/Kashgari/issues/310](https://github.com/BrikerMan/Kashgari/issues/310) but did not find the same one.
- [ ] I have read the [documents](https://kashgari.bmio.net)

## Environment

- OS [e.g. Mac OS, Linux]: win10
- Python Version: 3.62
- requirements.txt:
numpy==1.16.4
scikit-learn>=0.21.1
h5py
keras-bert>=0.50.0
keras-gpt-2>=0.8.0
gensim>=3.5.0
seqeval==0.0.10
pandas>=0.23.0
tensorflow-gpu=1.15.0

```txt
```

## Question
        训练分词，crf-loss出现负数，Kashgari版本为1.0.0，无论是keras-contrib, 还是你的CRF, 或者是其他github上的项目，都会出现损失为loss的情况，bert-embedding尤其明显，请问这可能是什么原因？

89/89 [==============================] - 231s 3s/step - loss: 0.0026 - accuracy: 1.0000 - val_loss: 124.0081 - val_accuracy: 0.9999
Epoch 74/132

 1/89 [..............................] - ETA: 3:11 - loss: -0.0046 - accuracy: 1.0000
 2/89 [..............................] - ETA: 3:18 - loss: -0.0036 - accuracy: 1.0000
 3/89 [>.............................] - ETA: 3:14 - loss: -0.0041 - accuracy: 1.0000
 4/89 [>.............................] - ETA: 3:12 - loss: -0.0045 - accuracy: 1.0000
 5/89 [>.............................] - ETA: 3:11 - loss: -0.0044 - accuracy: 1.0000
 6/89 [=>............................] - ETA: 3:10 - loss: -0.0043 - accuracy: 1.0000
 7/89 [=>............................] - ETA: 3:07 - loss: -0.0040 - accuracy: 1.0000
 8/89 [=>............................] - ETA: 3:04 - loss: -0.0040 - accuracy: 1.0000
 9/89 [==>...........................] - ETA: 3:03 - loss: -0.0039 - accuracy: 1.0000
10/89 [==>...........................] - ETA: 3:02 - loss: -0.0037 - accuracy: 1.0000
